packages/near-fast-auth-signer-e2e-tests/tests/auth.spec.ts

```diff
@@ -75,3 +86,11 @@ test('should create account and login with passkeys', async ({ page }) => {
 
   await pm.getAppPage().isLoggedIn();
 });
+
+test.afterAll(async () => {
```

Comment:
Nice improvement! 

But I'm wondering if it's better to use `afterEach` instead of `afterAll` as the tests runs in parallel. I'm not sure what will happen when we have a big set of tests running in parallel and the effects of multiple workers. 

What do you think?

Reply:
Good suggestion, but I'm not sure how would you implement in a way that afterEach would work with deleting correct email. Currently, I'm pushing in a list and delete them all once all tests are done, which is simple. If we want to use afterEach, you need to implement in a way that which email belongs to which test and delete the email of that particular test. (More complicated logic involved)

Secondly I have done some research and turns out that `.afterAll`  works per projects where:

> In Playwright, the test.afterAll hook is designed to run once after all tests within a particular project are completed. If you have multiple projects defined, the test.afterAll hook will run separately for each project after that project's tests are finished. It does not run once after all projects have completed.

> Running 3 projects:
>   - chromium
>   - firefox
>   - webkit
> 
> chromium
>   Before all tests
>   Running test
>   After all tests
> 
> firefox
>   Before all tests
>   Running test
>   After all tests
> 
> webkit
>   Before all tests
>   Running test
>   After all tests

So I think current approach is simple and does not compromise the performance. However if you could suggest implementation, feel free to do so.